docs/housing-counselor-data.md

```diff
@@ -0,0 +1,163 @@
+# Housing Counselor Data Processing
+
+The [Find a Housing Counselor tool](https://www.consumerfinance.gov/find-a-housing-counselor/) gets its data from the US Department of Housing and Urban Development. HUD provides an API to their list of approved counseling agencies.
+
+We generate the JSON data for the Find a Housing Counselor tool using the `hud_generate_json` management command. This page describes how that command works to transform HUD data to the JSON file format we use for the FAHC tool, including the following steps:
+  1. fetch agency listings from HUD
+  2. clean the results
+  3. fill in any missing latitude and longitude values
+  4. create files of the 10 nearest results for each US ZIP code
+
+
+## Fetch agency listings
+
+(_in `fetcher.py`_)
+
+Request every housing counselor in HUD's database with this request:
+
+```
+https://data.hud.gov/Housing_Counselor/searchByLocation?Lat=38.8951&Long=-77.0367&Distance=5000
+```
+
+It returns thousands of results like this:
+
+```json
+    {
```

Comment:
Without the context of the start of the file, the indentation in this example is unnecessary and could be removed.